packages/manager/apps/nasha-react/src/pages/index.scss

```diff
@@ -0,0 +1,64 @@
+ul {
```

Comment:
@stif59100 can you be more specific in the selector, add a class before or an id ?